src/App.tsx

```diff
@@ -1,18 +1,24 @@
 import React, { useEffect, useState } from 'react';
 import './App.css';
+import TodoList from './TodoList';
 
 interface Todo {
   id: number;
   title: string;
 }
-
 function App() {
   const [items, setItems] = useState<Todo[]>([]);
 
   useEffect(() => {
     fetch('https://jsonplaceholder.typicode.com/todos')
       .then((response) => response.json())
-      .then(setItems);
+      .then((data) => {
+        const todos = data.reduce((acc: Todo[], obj: Todo) => {
+          return [...acc, { id: obj.id, title: obj.title }];
```

Comment:
The format does not match REG-5 requirements.

Current: array of objects
Expected: array of arrays